recipes/platform.converters/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformConvertersConan(ConanFile):
+    name = "platform.converters"
```

Comment:
The name is a bit out of is used in Conan Center, but I'm fine as it's already used on nuget with the same name.